frontend/src/_components/OrganizationManager/EditOrganization.jsx

```diff
@@ -48,6 +48,12 @@ export const EditOrganization = ({ showEditOrg, setShowEditOrg, currentValue })
     setNewOrgName(e.target.value);
   };
 
+  const handleKeyDown = (e) => {
```

Comment:
```suggestion
  const handleKeyDown = (e) => {
    if (e.key === 'Enter') {
      e.preventDefault();
      editOrganization();
    }
  };
```
Instead of using the keyCode property, it's recommended to use the key property to handle key events. The keyCode property is deprecated, and the key property provides a more consistent and readable way to handle specific keys.